client/me/purchases/purchase-item/index.jsx

```diff
@@ -505,6 +506,12 @@ class PurchaseItem extends Component {
 				);
 			}
 
+			if ( purchase.payment.type === 'upi' ) {
+				return (
+					<img src={ upiImage } alt={ purchase.payment.type } className="purchase-item__upi" />
```

Comment:
I noticed that this `purchase-item__upi` CSS class is not defined anywhere. The other similar classes for the other payment methods mostly set a`width` and `height`. We are already inheriting the properties from the parent class, and the logo itself looks good in different screen sizes, albeit maybe a bit small. 

I think we can remove this class name prop.

Reply:
That makes sense to me.  If we decide we need to style it differently in the future, we can always add a class name at that time.